test/Microsoft.AspNet.Identity.EntityFramework.Test/CustomPocoTest.cs

```diff
@@ -46,7 +46,7 @@ public class CustomDbContext<TKey> : DbContext where TKey : IEquatable<TKey>
 
         [TestPriority(-1000)]
         [ConditionalFact]
-        [FrameworkSkipCondition(RuntimeFrameworks.Mono)]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono)][OSSkipCondition(OperatingSystems.Linux)][OSSkipCondition(OperatingSystems.MacOSX)]
```

Comment:
Do you actually mean we will never want to run them on Linux and Mac? Or is it just a temporary limitation?